index.html

```diff
@@ -6,6 +6,6 @@
     <title>Document</title>
 </head>
 <body>
-    
+    <h1>Heading</h1>
```

Comment:
This should want h2 tag and content should want change as like this 'Hello JavaScript'